third_party/terraform/tests/resource_logging_metric_test.go

```diff
@@ -0,0 +1,53 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccLoggingMetricUpdate(t *testing.T) {
+	t.Parallel()
+
+	suffix := acctest.RandString(10)
+	filter := "resource.type=gae_app AND severity>=ERROR"
+	updatedFilter := "resource.type=gae_app AND severity=ERROR"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckLoggingMetricDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccLoggingMetric_ForUpdate(suffix, filter),
+			},
+			{
+				ResourceName:      "google_logging_metric.logging_metric",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccLoggingMetric_ForUpdate(suffix, updatedFilter),
+			},
+			{
+				ResourceName:      "google_logging_metric.logging_metric",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccLoggingMetric_ForUpdate(suffix string, filter string) string {
```

Comment:
```suggestion
func testAccLoggingMetric_update(suffix string, filter string) string {
```

When configs are 1:1 with a test case, generally we match the test name exactly w/ the first char lowercased.